events/wmt21.md

```diff
@@ -209,3 +209,51 @@ Thursday, November 11, 2021
 | 15:00 – 15:15 | [***Subword Mapping and Anchoring across Languages***](https://arxiv.org/pdf/2109.04556.pdf) <br>Giorgos Vernikos and Andrei Popescu-Belis |
 | 15:15 – 15:30 | [***Uncertainty-Aware Machine Translation Evaluation***](https://arxiv.org/pdf/2109.06352.pdf) <br>Taisiya Glushkova, Chrysoula Zerva, Ricardo Rei and André F. T. Martins |
 | 15:30 – 15:45 | [***Sometimes We Want Ungrammatical Translations***](https://aclanthology.org/2021.findings-emnlp.275.pdf) <br>Prasanna Parasarathi |
+
+## Results
+
+#### Full results of the shared tasks: [***Findings of the 2021 Conference on Machine Translation (WMT21)***](https://statmt.org/wmt21/pdf/2021.wmt-1.1.pdf)
+
+### News Translation
+
+#### → English (segment rating with document context)
+
+| Language pair | Average score | Average z-score (standard score) | System |
+| --- | --- | --- | --- |
+| Czech→English | 77.8 | 0.111 | `Facebook-AI` |
```

Comment:
I'd put a space around the arrow.

Maybe we can remove "English" since it's in the section title?